docs/concepts/endpoint.md

```diff
@@ -1,117 +1,137 @@
 ---
```

Comment:
I read your new effort, it became better than before.
One thing that I can note is that in concept pages we don't need detailed examples, we only want to show‌ (not practical detailed working) what that concept is and how that works, maybe with some simple examples. 
Please note that this comment doesn't mean you should change something, but I thought it might help you if you want to improve the quality of your writing.

Reply:
noted